### ISSUE TITLE
8.2.2 support

### ISSUE THREAD

**Issue by egandro** [brunokc/pve-kernel-builder#15]
are you interssted in bringing this to a more recent version?

proxmox 8.2.2 with 6.8.4-x-pve has some issues / crashes

I would love to bringe back the pve-edge kernel support but I need some help

**Comment by brunokc** [comment]
What do you mean by 'pve-edge'?

I'm running kernel 6.8.4-3-pve with the relaxable RMRRs enabled. Haven't seen a crash yet. What kind of crashes are you seeing? Have you reported them to Proxmox?

There isn't much that this project can do for Proxmox kernels 6.2+ as they already contain the relaxable RMRR patch included here. Unless you're referring to a different set of patches...

**Comment by egandro** [comment]
> What do you mean by 'pve-edge'?

https://github.com/fabianishere/pve-edge-kernel

> I'm running kernel 6.8.4-3-pve with the relaxable RMRRs enabled. Haven't seen a crash yet. What kind of crashes are you seeing? Have you reported them to Proxmox?

Yes. There are several severe issues. It crashes on Xeons, Epyc, Intel... there is no real pattern. I debugged it by removing the pve patches. It looks like there is an Ubuntu kernel issue. I debugged 5 days on that to come to this conclusion.

Zero reaction from Proxmox. Several threads in their forums. Zero help where ever I asked in Discord etc. for "just join forces" for our own edge kernel. Now I ask you. The original author of pve-edge has no time to maintain this any longer.

> There isn't much that this project can do for Proxmox kernels 6.2+ as they already contain the relaxable RMRR patch included here. Unless you're referring to a different set of patches...

I would love to have a 6.8.x or a 6.9.x version with ZFS. The Proxmox patches are tiny and not really essential. The ZFS version needs to be aligend with the ZFS utils that are installed on Proxmox.
